.circleci/config.yml

```diff
@@ -591,13 +628,253 @@ workflows:
     jobs:
       - build
       - unittest
+      - pre_integrationtest
+      - integrationtest:
+          name: apigateway
+          testmodule: aws-android-sdk-apigateway-test
+          requires:
+            - pre_integrationtest          
+          filters:
+            branches:
+              only:
+                - master
+                - develop
+                - runintegrationtest
+      - integrationtest:
+          name: autoscaling
+          testmodule: aws-android-sdk-autoscaling-test
+          requires:
+            - pre_integrationtest    
+          filters:
+            branches:
+              only:
+                - master
+                - develop
+                - runintegrationtest
+      - integrationtest:
+          name: cloudwatch
+          testmodule: aws-android-sdk-cloudwatch-test
+          requires:
+            - pre_integrationtest    
+          filters:
+            branches:
+              only:
+                - master
+                - develop
+                - runintegrationtest
       - integrationtest:
+          name: comprehend
+          testmodule: aws-android-sdk-comprehend-test
+          requires:
+            - pre_integrationtest    
+          filters:
+            branches:
+              only:
+                - master
+                - develop
+                - runintegrationtest
+ 
+      - integrationtest:
+          name: ddb-mapper
+          testmodule: aws-android-sdk-ddb-mapper-test
+          requires:
+            - pre_integrationtest    
+          filters:
+            branches:
+              only:
+                - master
+                - develop
+                - runintegrationtest 
+      - integrationtest:
+          name: ddb
+          testmodule: aws-android-sdk-ddb-test
+          requires:
+            - pre_integrationtest    
           filters:
             branches:
               only:
                 - master
                 - develop
+                - runintegrationtest  
+      - integrationtest:
```

Comment:
We need to add kinesisvideo and kinesisvideoarchivedmedia to the list

Reply:
done